Poker.BE/Poker.BE.Domain/Game/Pot.cs

```diff
@@ -8,11 +8,46 @@ namespace Poker.BE.Domain.Game
 {
     public class Pot
     {
+        #region Fields
+        private Pot partialPot;
+        private Pot basePot;
+        private int value;
+        private int amountToClaim;
+        private List<Player> playersClaimPot;
+        #endregion
+
         #region Properties
-        public Pot PartialPot { get; set; }
+        public Pot PartialPot { get { return this.partialPot; } set { this.partialPot = value; } }
+        public Pot BasePot { get { return this.basePot; } set { this.basePot = value; } }
+
+        public int Value { get { return this.value; } set { this.value = value; } }
+        public int AmountToClaim { get { return this.amountToClaim; } set { this.amountToClaim = value; } }
+
+        public List<Player> PlayersClaimPot { get { return this.playersClaimPot; } set { this.playersClaimPot = value; } }
         #endregion
 
-        // TODO: complete - set team member to do this
+        #region Constructors
+        public Pot(Pot basePot)
```

Comment:
are you sure that using this constructor as only constructors is the best design? saw many initialization like "new Pot(null);" probably more than actually using the basePot parameter.
maybe you should split it to 2 constructors, one of them should be empty constructor that assign null to basepot? or extract subclass for partials?